client/state/themes/actions/trending-themes.js

```diff
@@ -35,7 +35,7 @@ export function getTrendingThemes( filter ) {
 			apiVersion: '1.1',
```

Comment:
`apiVersion` is redundant, and the `query` object could be inlined into the `.get` call.

Reply:
👍  Cleaned up further in 32bee26a52956d71780c08b057576fd1ed5dd0c9.